src/firestore.ts

```diff
@@ -0,0 +1,195 @@
+import firebase from './firebase';
```

Comment:
See all comments from functions version of this file. Also is it necessary to have this file in both functions and src? Anyway to externalize it?

Reply:
Unfortunately I don't think so without making some external library, the error is "can't import from outside this src directory". They are also slightly different files since the `Timestamp` type is `FirebaseFirestore.Timestamp` in `functions` and `firebase.firestore.Timestamp` in `src`